<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageTab</class>
 <widget class="QWidget" name="ImageTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Image Contents</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="_matrixLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Matrix:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Kst::MatrixSelector" name="_matrix" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_imageTypeGroup">
     <property name="title">
      <string>Image Type</string>
     </property>
     <layout class="QGridLayout" name="_6">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="_colorOnly">
        <property name="text">
         <string>Color m&amp;ap</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="_contourOnly">
        <property name="text">
         <string>Con&amp;tour map</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="_colorAndContour">
        <property name="text">
         <string>&amp;Color map and contour map</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_colorMapGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Color Map Parameters</string>
     </property>
     <layout class="QGridLayout" name="_7">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item row="1" column="2" colspan="2">
       <layout class="QHBoxLayout" name="_8">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel2">
          <property name="text">
           <string>&amp;Upper:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>_upperThreshold</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_upperThreshold">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="_9">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel2_4">
          <property name="text">
           <string>Threshold - &amp;Lower:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>_lowerThreshold</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_lowerThreshold">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="_10">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="_smartThreshold">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Smart</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Percent&amp;ile:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>_smartThresholdValue</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="_smartThresholdValue">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>45.000000000000000</double>
          </property>
          <property name="value">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>41</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="Kst::ColorPalette" name="_colorPalette" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="_autoThreshold">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Max&amp;/Min</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QCheckBox" name="_realTimeAutoThreshold">
        <property name="text">
         <string>&amp;Real-time auto threshold</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_contourMapGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Contour Map Parameters</string>
     </property>
     <layout class="QGridLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel2_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Co&amp;lor:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>_contourColor</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Kst::ColorButton" name="_contourColor">
          <property name="text">
           <string/>
          </property>
          <property name="color" stdset="0">
           <color>
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Num&amp;ber of contour levels:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>_numContourLines</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="_numContourLines">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="_4">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Weight:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>_contourWeight</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="_contourWeight">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="maximum">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1" colspan="2">
       <layout class="QHBoxLayout" name="_5">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="_useVariableWeight">
          <property name="text">
           <string>Use &amp;variable line weight</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>76</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Kst::CurvePlacement" name="_curvePlacement" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Kst::ColorButton</class>
   <extends>QToolButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::CurvePlacement</class>
   <extends>QWidget</extends>
   <header>curveplacement.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::ColorPalette</class>
   <extends></extends>
   <header>colorpalette.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::MatrixSelector</class>
   <extends></extends>
   <header>matrixselector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
